This turns off dynamically loading the statx() symbol, which has the same effect as reverting
https://github.com/openjdk/jdk17u/commit/184fac8af61633ccf833eda53183a27da8efb0f7 ,
which was fixing https://bugs.openjdk.org/browse/JDK-8316304 ,
because on some devices, for some reason it causes launching any JAR to crash the Java runtime.

Fixes https://github.com/termux/termux-packages/issues/24651

--- a/src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c
+++ b/src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c
@@ -434,7 +438,7 @@ Java_sun_nio_fs_UnixNativeDispatcher_init(JNIEnv* env, jclass this)
 #ifdef _DARWIN_FEATURE_64_BIT_INODE
     capabilities |= sun_nio_fs_UnixNativeDispatcher_SUPPORTS_BIRTHTIME;
 #endif
-#if defined(__linux__)
+#if defined(__linux__) && !defined(__ANDROID__)
     my_statx_func = (statx_func*) dlsym(RTLD_DEFAULT, "statx");
     if (my_statx_func != NULL) {
         capabilities |= sun_nio_fs_UnixNativeDispatcher_SUPPORTS_BIRTHTIME;
